neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -25,6 +25,15 @@ internal GasToken()
         {
         }
 
+        internal override bool Initialize(ApplicationEngine engine)
+        {
+            if (!base.Initialize(engine)) return false;
+            if (TotalSupply(engine.Snapshot) != BigInteger.Zero) return false;
+            UInt160 account = Contract.CreateMultiSigRedeemScript(Blockchain.StandbyValidators.Length / 2 + 1, Blockchain.StandbyValidators).ToScriptHash();
+            Mint(engine, account, 30_000_000 * Factor);
```

Comment:
Maybe we can provide this 30M as parameter in order to make the UT pass in any given scenario.